app/src/main/java/com/example/ohthmhyh/CreateHabitEvent.java

```diff
@@ -305,7 +309,16 @@ public void final_create_habit(View view) {
         //TODO
         //Get user info
         //Get a habit from user
-        HabitEvent habitEvent=new HabitEvent(habit,comment,address,bitmap,-1, habiteventlist.nextUPID());
+
+        HabitEvent habitEvent;
+        if(address == null){
+            habitEvent=new HabitEvent(habit,comment,-1, -1
+                    , bitmap,-1, habiteventlist.nextUPID());
```

Comment:
Up in line 140, we compare the lat and lon to null, but here we are setting the lat and lon to -1. Should we set the lat and lon here to null?

Reply:
That would be just fine too. I just put in some dummy values for now because I thought this stuff was being rewritten anyway

Reply:
Okay, just for consistency, could you make this change? I think it will also change the constructor of the HabitEvent to take in `Double`s instead of `double`s for lat and lon.

Reply:
Say no more; consider it done

Reply:
done